### DISCUSSION TITLE
re-locate Tile3DLayer

### DISCUSSION THREAD

**Discussion by cozybim** [visgl/deck.gl#9152] [Q&A]
When loading data with Tile3DLayer, I want to change the position of the entire data on the map. I have tried various methods, such as using modelMatrix, accessing each sub-tile and changing the modelMatrix, adding a transformMatrix field in the tileset.json definition, using coordinateSystem: COORDINATE_SYSTEM.LNGLAT_OFFSETS and set value to coordinateOrigin... However, the data position still does not have any change. Please help with a solution to move the entire 3D Tile to a new location. Thank you

**Comment by ibgreen** [comment]
@cozybim Thanks for reporting.

- Making modelMatrix prop work for the Tile3DLayer is a long standing wish,
- I haven't looked at this for a long time, but I have been under the impression that the Tile3DLoader uses the deck.gl modelMatrix prop to position each tile, effectively "taking this prop away from the user".
- If true, it may be possible to just add some code in Tile3DLayer that multiplies the user provided matrix with the one in the tile.

@belom88

**Comment by cozybim** [comment]
@belom88 Thank you for providing the information. Therefore, the best way is to convert the coordinates of the entire 3D Tile data before loading it into Deck GL
